packages/fluent-theme/src/components/activity/CopilotMessageHeader.module.css

```diff
@@ -38,5 +39,6 @@
   font-size: var(--webchat-fontSizeBase100);
   height: 20px;
   line-height: var(--webchat-lineHeightBase100);
-  padding-inline: 4px;
+  padding-inline: var(--webchat-spacingHorizontalXS);
+  flex: none;
```

Comment:
Sort.

Reply:
Both should work. I prefer flex: auto/none